vita3k/renderer/src/texture/pvrt-dec.cpp

```diff
@@ -55,19 +55,19 @@ struct Pixel32 {
 struct Pixel128S {
     int32_t red, green, blue, alpha;
 
-    Pixel128S operator+(const int32_t uiFactor) {
+    Pixel128S operator+(const int32_t uiFactor) const {
```

Comment:
This file is taken from PowerVR SDK, so it would be better to send your fixes to upstream